phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -583,6 +583,7 @@ private static int getReservedQualifier(byte[] bytes, int offset, int length) {
     PName getName();
     PName getSchemaName();
     PName getTableName();
+    PName getPhysicalTableNameColumnInSyscat();
```

Comment:
getPhysicalNames makes a point of returning a list of PNames to leave the interface open for views that span multiple tables (not currently supported but long on the wishlist). Does anything in this PR, such as getPhysicalTableNameColumnInSyscat returning a single String, prevent us from having multi-table views later?